src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -57,20 +56,19 @@ public void tearDown() {
     }
 
     @Test
-    public void generates_and_saves_pdf() throws IOException {
+    public void generates_and_saves_zip_file() throws IOException {
         UUID id = service.send(SampleData.letter(), SERVICE_NAME);
+
         Letter result = letterRepository.findOne(id);
-        DataSource dataSource = new ByteArrayDataSource(new ByteArrayInputStream(result.getFileContent()));
-        PreflightParser pdfParser = new PreflightParser(dataSource);
-        pdfParser.parse();
-        PreflightDocument document = pdfParser.getPreflightDocument();
-        // This will throw an exception if the file format is invalid,
-        // but ignores more minor errors such as missing metadata.
-        document.validate();
+
+        ZipInputStream zip = new ZipInputStream(new ByteArrayInputStream(result.getFileContent()));
```

Comment:
We could keep the existing checks and validate that it unzips to a valid pdf, just by wrapping the ByteArrayDataSource constructor parameter in a ZipInputStream.